daprdocs/content/en/operations/hosting/self-hosted/self-hosted-no-docker.md

```diff
@@ -6,36 +6,49 @@ weight: 30000
 description: "How to deploy and run Dapr in self-hosted mode without Docker installed on the local machine"
 ---
 
-This article provides guidance on running Dapr in self-hosted mode without Docker.
-
 ## Prerequisites
 
-- [Dapr CLI]({{< ref "install-dapr-selfhost.md#installing-dapr-cli" >}})
+- [Install the Dapr CLI]({{< ref "install-dapr-selfhost.md#installing-dapr-cli" >}})
 
 ## Initialize Dapr without containers
 
-The Dapr CLI provides an option to initialize Dapr using slim init, without the default creation of a development environment which has a dependency on Docker. To initialize Dapr with slim init, after installing the Dapr CLI use the following command:
+The Dapr CLI provides an option to initialize Dapr using slim init, without the default creation of a development environment with a dependency on Docker. Once you've installed the Dapr CLI, slim init Dapr with the following command:
```

Comment:
This sentence does not read right anymore. 
"Once you've installed the Dapr CLI, slim init Dapr with the following command:"